Kernel/Scheduler.cpp

```diff
@@ -48,12 +50,26 @@ Thread* g_last_fpu_thread;
 Thread* g_finalizer;
 static Process* s_colonel_process;
 u64 g_uptime;
-static u64 s_beep_timeout;
+static u64 s_next_timer_due;
 
 struct TaskRedirectionData {
     u16 selector;
     TSS32 tss;
 };
+
+struct Timer {
+    u64 expires;
+    void (*callback)();
+    inline bool operator<(const Timer& rhs)
```

Comment:
`inline` is meaningless in this context. Also please mark it const:
```suggestion
    bool operator<(const Timer& rhs) const
```